### ISSUE TITLE
Unable to run precompilled GodotSteam Multiplayer (opengl error)

### ISSUE THREAD

**Issue by alexxdann** [GodotSteam/MultiplayerPeer#12]
**Describe the bug**
Can't run Godot 4.3 - Steamworks 1.60 - GodotSteam 4.11 MultiplayerPeer [(link)](https://github.com/GodotSteam/MultiPlayerPeer/releases/tag/v4.11-mp)

**To Reproduce**
1. Open godotsteam.multiplayer.43.editor.windows.64.exe
![image](https://github.com/user-attachments/assets/ba4c0149-883e-4d9b-a885-0ba99c247a53)

**Expected behavior**
Open godot editor with custom build.

**Screenshots**
Run common godot from console (Successful):
![image](https://github.com/user-attachments/assets/0b0e2e8d-fe47-4640-8231-68ea3fd1bc03)

Run precompilled godotstean from console (Issue):
![image](https://github.com/user-attachments/assets/75d71e59-69cd-4d16-b3df-db2e07eda077)

**OS:**
Windows Version - Windows 10 (64 bit)
2D Driver Version - 8.1.1.1634
Direct3D® Version - 9.14.10.01494
OpenGL® Version - 29.20.11000.14830
Vulkan™ Driver Version - 2.0.179
Vulkan™ API Version - 1.2.170
GPU - AMD Radeon R7 Graphics - Primary/Integrated
VRAM - 1024 MB - DDR4 933 MHz
Graphics Card Manufacturer - Powered by AMD
Graphics Chipset - AMD Radeon R7 Graphics
OpenGL® API Version - 4.6
OpenCL™ API Version - 2.0

**Version of Godot:**
4.3

**Version of GodotSteam:**
precompilled 4.11

**Comment by Gramps** [comment]
Hey there!  Hmm, not sure why that wouldn't work but I do not have Windows to test this.  However, I think this would have come up before as it has been out for a while and other people use it. Might have to do with Angle?

**Comment by alexxdann** [comment]
> Hey there! Hmm, not sure why that wouldn't work but I do not have Windows to test this. However, I think this would have come up before as it has been out for a while and other people use it. Might have to do with Angle?

The point is that this's working for usual godot, so I assume that this's should work for custom build as well. Unless if godotsteam touched some rendering stuff.

**Comment by Gramps** [comment]
I haven't looked too closely at the newest stuff they included in Godot like Direct 12 or Angle; I'm not totally sure what the Angle stuff even is, honestly,  I do know none of our Github Actions builds use any of it since it requires a bunch of extra libraries.

I've seen this error on much older laptops but that clearly isn't the case here.  And, as you mention, GodotSteam doesn't really affect anything on the rendering side; it is just a layer on top of Godot. The only difference is we probably do not compile it the same way at all.  I do know Godot compiles their Windows builds with MinGW, which we cannot do, and they include extra libraries like Direct 12, etc.

You could try self-compiling it with a mix of their instructions and ours.  At some point, I will get around to re-writing those GA build files but time is incredibly limited and they are at the bottom of a very long list.

**Comment by alexxdann** [comment]
Sure, I'll try to compile it, maybe on my env it will fit.
Btw If you mean this [issue](https://github.com/GodotSteam/MultiplayerPeer/issues/10) I think it's 100% related since it also "amd radeon r7 graphics".
In that case this issue could be marked as dublicated.

**Comment by Gramps** [comment]
Awesome, let me know how that goes!  Again, _do not_ use MinGW or it will crash.

I may be related and probably specific to Windows.  My laptop uses AMD Radeon and I can run it fine, however, I use Linux.

**Comment by Gramps** [comment]
Yeah, if you look at Godot's documentation, Angle has to be added separately. They include it during their build process; however, we have not updated our Github Actions workflows to include it yet and I have no timeline on when that might happen.

Did you try following their instructions on how to add it?

**Comment by alexxdann** [comment]
> Did you try following their instructions on how to add it?
Yep, once I'll able to build godot+godotsteam - opengl issue should be resolved.

**Comment by Gramps** [comment]
There is a sort of order I do these things in and am partially through with the Server updates. Given the limited time and funding, things move a little slowly.  That being said, if someone wants to patch the Github Actions files, we can just automate the process of building the updated Angle-enabled versions.  That being said, this is slated to be added to the next line of patches after Servers are done; which will then require backporting to Servers.
